src/containers/ForgotPassword.jsx

```diff
@@ -37,52 +38,47 @@ class ForgotPassword extends Component {
     } = this.props;
     const { open } = this.state;
     return (
-      <div className="container">
-        <div className="subContainer">
-          <Scenic />
-        </div>
-        <div className="subContainer">
-          <div className="containerPadding">
-            <h1>
-              Forgot Password
-            </h1>
-            <form onSubmit={handleSubmit(this.onSubmit.bind(this))}>
-              <Field
-                label="Email"
-                name="email"
-                type="text"
-                component={TextInput}
-              />
+      <PageContainer>
+        <div className="containerPadding">
+          <h1>
+            Forgot Password
+          </h1>
+          <form onSubmit={handleSubmit(this.onSubmit.bind(this))}>
```

Comment:
is this function supposed to work? clicking submit doesn't do anything (nor does it return any console errors)

Reply:
Forgot password is actually the feature im currently working on right now and isn't included in this PR